sdk/inc/azure/iot/az_iot_pnp_client.h

```diff
@@ -373,66 +373,67 @@ AZ_NODISCARD az_result az_iot_pnp_client_commands_response_get_publish_topic(
 
 /**
  *
- * Twin APIs
+ * Property APIs
  *
  */
 
 /**
- * @brief The MQTT topic filter to subscribe to twin operation responses.
- * @note Twin MQTT Publish messages will have QoS At most once (0).
+ * @brief The MQTT topic filter to subscribe to property operation responses.
+ * @note Property MQTT Publish messages will have QoS At most once (0).
  */
-#define AZ_IOT_PNP_CLIENT_TWIN_RESPONSE_SUBSCRIBE_TOPIC "$iothub/twin/res/#"
+#define AZ_IOT_PNP_CLIENT_PROPERTY_RESPONSE_SUBSCRIBE_TOPIC "$iothub/twin/res/#"
 
 /**
- * @brief Gets the MQTT topic filter to subscribe to twin desired property changes.
- * @note Twin MQTT Publish messages will have QoS At most once (0).
+ * @brief Gets the MQTT topic filter to subscribe to desired property changes.
+ * @note Property MQTT Publish messages will have QoS At most once (0).
  */
-#define AZ_IOT_PNP_CLIENT_TWIN_PATCH_SUBSCRIBE_TOPIC "$iothub/twin/PATCH/properties/desired/#"
+#define AZ_IOT_PNP_CLIENT_PROPERTY_PATCH_SUBSCRIBE_TOPIC "$iothub/twin/PATCH/properties/desired/#"
 
 /**
- * @brief Twin response type.
+ * @brief Property response type.
  *
  */
 typedef enum
 {
-  AZ_IOT_PNP_CLIENT_TWIN_RESPONSE_TYPE_GET
-  = AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_GET, /**< A response from a twin "GET" request. */
-  AZ_IOT_PNP_CLIENT_TWIN_RESPONSE_TYPE_DESIRED_PROPERTIES
+  AZ_IOT_PNP_CLIENT_PROPERTY_RESPONSE_TYPE_GET
+  = AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_GET, /**< A response from a property "GET" request. */
+  AZ_IOT_PNP_CLIENT_PROPERTY_RESPONSE_TYPE_DESIRED_PROPERTIES
   = AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_DESIRED_PROPERTIES, /**< A "PATCH" response with a payload
                                                                 containing desired properties. */
-  AZ_IOT_PNP_CLIENT_TWIN_RESPONSE_TYPE_REPORTED_PROPERTIES
+  AZ_IOT_PNP_CLIENT_PROPERTY_RESPONSE_TYPE_REPORTED_PROPERTIES
   = AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_REPORTED_PROPERTIES, /**< A response with the result of the
                                                                  earlier reported properties. */
-} az_iot_pnp_client_twin_response_type;
+} az_iot_pnp_client_property_response_type;
 
 /**
- * @brief Twin response.
+ * @brief Property response.
  *
  */
 typedef struct
 {
-  az_iot_pnp_client_twin_response_type response_type; /**< Twin response type. */
+  az_iot_pnp_client_property_response_type response_type; /**< Property response type. */
   az_iot_status status; /**< The operation status. */
   az_span
       request_id; /**< Request ID matches the ID specified when issuing a Get or Patch command. */
-  az_span version; /**< The Twin object version.
-                    * @note This is only returned when
-                    * response_type==AZ_IOT_CLIENT_TWIN_RESPONSE_TYPE_DESIRED_PROPERTIES
-                    * or
-                    * response_type==AZ_IOT_CLIENT_TWIN_RESPONSE_TYPE_REPORTED_PROPERTIES. */
-} az_iot_pnp_client_twin_response;
+  az_span
+      version; /**< The property object version.
```

Comment:
Nit - it's always going to be returned because it's in a struct.  I think you mean it's only going to be specified (non az_span_null) in this case??